### PR TITLE
feat: use i18n on enter email

### PR DESCRIPTION
**Pull request by danielgent**

### CODE REVIEW

File: src/utils/components/LoginWithCodePage.tsx [hunk 1]
```diff
@@ -2,27 +2,39 @@ import type { FC } from "hono/jsx";
...
 
 type Props = {
   error?: string;
   vendorSettings: VendorSettings;
 };
 
+// this page is called enter-email on login2... maybe we should copy those page names
 const LoginWithCodePage: FC<Props> = ({ error, vendorSettings }) => {
+  // TODO - get this from auth0header
+  const sendType = "code";
```
Comment (danielgent) [inline src/utils/components/LoginWithCodePage.tsx:15]: I'll do these on future PRs :+1:

Comment (danielgent) [reply src/utils/components/LoginWithCodePage.tsx:15]: Every PR they'll be a bit closer to login2 :smile:

File: src/utils/components/LoginWithCodePage.tsx [hunk 2]
```diff
@@ -2,27 +2,39 @@ import type { FC } from "hono/jsx";
...
 import Button from "./Button";
 import { VendorSettings } from "../../types";
+import i18next from "i18next";
 
 type Props = {
   error?: string;
   vendorSettings: VendorSettings;
 };
 
+// this page is called enter-email on login2... maybe we should copy those page names
```
Comment (danielgent) [inline src/utils/components/LoginWithCodePage.tsx:12]: :eyes: this is definitely the source of some confusion

Comment (markusahlstrand) [reply src/utils/components/LoginWithCodePage.tsx:12]: Yepp, think it's safer to keep this consistent with auth0

**Review by markusahlstrand — APPROVED** [review]
LGTM

File: src/utils/components/LoginWithCodePage.tsx (hunk 2 above)
Comment (danielgent) [reply src/utils/components/LoginWithCodePage.tsx:12]: @markusahlstrand I was referring to login2 *BUT* I'll copy auth0's universal auth at some point

I can never remember where that setting is that changes the Auth0 flow to be email first step, code second...

I don't want to change our main client of course so I should experiment on a new one

**Pull request merged**